netlify.toml

```diff
@@ -1,16 +1,6 @@
 [build]
   publish = "public"
-  command = "cd themes/docsy && git submodule update -f --init && npm install && cd ../.. && hugo --gc --minify"
+  command = "npm run build:preview"
 
-[context.deploy-preview.environment]
-  HUGO_VERSION = "0.119.0"
-  NODE_VERSION = "18"
-
-[context.production.environment]
-  HUGO_VERSION = "0.119.0"
-  HUGO_ENV = "production"
-  NODE_VERSION = "18"
-
-[context.branch-deploy.environment]
-  HUGO_VERSION = "0.119.0"
-  NODE_VERSION = "18"
```

Comment:
@chalin Won't removing this prevent the old branches like https://github.com/kubeflow/kubeflow/tree/v1.7-branch, from being deployed, because we achieve this by using a CNAME record from:

- `v1-7-branch.kubeflow.org` -> `v1-7-branch--competent-brattain-de2d6d.netlify.app`

I don't have access to the Netelify dashboard, so I can't see where those are being deployed from.

Reply:
@thesuperzapper - from my understanding of how Netlify works, unless you back port this change to an older v1.x branch, the builds of other branches won't be affected.

For example, v1.7 has it's own `netlify.toml` file, https://github.com/kubeflow/website/blob/v1.7-branch/netlify.toml, with it's own env var values and commands:
https://github.com/kubeflow/website/blob/b48a664062876fdc54aee9c831d0032924734a1e/netlify.toml#L1-L16

But, I could be wrong.

Reply:
@james-jwu or @zijianjoy do you have access to Netelify for the Kubeflow website, so we can confirm how it is configured?

We probably also need to give access in some way to the CNCF for continuity purposes.